packages/tools/src/tools/WindowLevelTool.ts

```diff
@@ -190,12 +191,7 @@ class WindowLevelTool extends BaseTool {
 
     const ratio = imageDynamicRange / DEFAULT_IMAGE_DYNAMIC_RANGE;
 
-    let multiplier = DEFAULT_MULTIPLIER;
-
-    if (ratio > 1) {
-      multiplier = Math.round(ratio);
-    }
-    return multiplier;
+    return ratio > 1 ? Math.round(ratio) : ratio;
```

Comment:
Apologies for the delay, I've been occupied with various tasks. I believe treating non-scaled PT like any other image is a good idea and I'm completely fine with it. However, I noticed that we are also modifying the `multiplyer` for all images. Is there a specific reason for this change?

Reply:
Thanks for the feedback.

For images with small dynamic range (which result in `ratio < 1` in this code), the previous multiplier was too big. See the before.mov video in my original post which shows that the Window Tool is basically unusable for images with small dynamic range. This is the case for all modalities, not just PT, that's why I changed it here.

With this change the multiplier is always based on the calculated dynamic range, and for `ratio > 1` the behavior is the same as before.

Reply:
Ok makes sense, I pushed some test fixes and will merge after